### DISCUSSION TITLE
Welcome to Raspberry Discussions!

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/Raspberry#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by citizenhicks** [comment]
hi, this is eddie and i work with ai models, particularly in the reinforcement learning space for a while now. 

if i can i would love to contribute. i'm hoping, if we can build a scalable and reliable synthetic data fun-tuning pipeline, that could lead to many other fine tuning applications, not just reasoning.

**Reply by daveshap** [reply]
please share your portfolio or work and the team and I will take a look

**Reply by citizenhicks** [reply]
hi, check out my public github. that is a good summary of what i've been working with in the last couple of years. cheers!

**Reply by daveshap** [reply]
Sorry but I am not going to go digging through 17 repos to try and piece together what you've been working on. I'll need to see a bit more effort on your part if you want to get involved.

**Reply by citizenhicks** [reply]
that's fair, i've summarised it on my GitHub page: https://github.com/Arrabonae

**Comment by Allknowingroger** [comment]
When they said they've scaled the inference, is that increasing the number of steps it runs through the model? Or is it bigger hardware server?

Also-
"Jailbroke o1 and got the reasoning steps"
https://huggingface.co/posts/nisten/520824119529412

Reasoning Dataset
https://huggingface.co/datasets/Cyleux/skunk-reasoning-29k-sharegpt

**Reply by daveshap** [reply]
scaling inference, in this case, just means that it talks to itself longer.

**Reply by russellballestrini** [reply]
I know we suspect it's a single shot for the chain and solution but do you think it could be in a loop that does a simple validate (user agent)? Single shot best case otherwise loop?

**Reply by daveshap** [reply]
no, noam brown said it wasn't

**Comment by russellballestrini** [comment]
Welcome all, I'm a long time viewer of the channels on YouTube.

I'm glad to contribute! 

I plan to consume this model when its ready in my app, The code is here:

* https://github.com/russellballestrini/flask-socketio-llm-completions 

For my own currently researching and exploring ways build AI's for battleship.

The game file is here for the application if you want to see how it works:

* https://github.com/russellballestrini/flask-socketio-llm-completions/blob/main/research/activity29-battleship.yaml

**Comment by durapensa** [comment]
Hi, David here. I'm thinking that training data to teach the use of `<artifact>` and `<thinking>` tags (as well as others that may be useful like `<reflect>`), as Anthropic did for Claude 3.5, could be a useful adjunct to reasoning training datasets. Allowing, e.g., an end-user focused chatbot to present summaries of reasoning steps as ChatGPT-o1 preview does.
